### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by jnm2**
I saw a ton while adding and removing target frameworks and made notes to come back and fix them.

### CODE REVIEW

**Comment by jnm2** [comment]
Oh, that's stupid. I nailed a test and forgot to run tests locally.

File: src/NUnitFramework/tests/Internal/Results/TestResultFailureTests.cs [hunk 1]
```diff
@@ -202,7 +202,7 @@ public void SuiteResultXmlNodeIsFailure()
             Assert.NotNull(messageNode, "No <message> element found");
             Assert.AreEqual(TestResult.CHILD_ERRORS_MESSAGE, messageNode.Value);
 
-            TNode stacktraceNode = failureNode.SelectSingleNode("stacktrace");
+            TNode stacktraceNode = failureNode.SelectSingleNode("stack trace");
```
Comment (mikkelbu) [inline src/NUnitFramework/tests/Internal/Results/TestResultFailureTests.cs]: I know we have our own code representing XML, but we cannot have a element with a space in the name.

Comment (jnm2) [reply src/NUnitFramework/tests/Internal/Results/TestResultFailureTests.cs]: Thanks! I'm rereviewing too. I don't intend to make any behavioral changes. I'll blame almost programming for 21 hours straight. 🤨

**Comment by jnm2** [comment]
Alright, I believe all the changes are good and purely cosmetic.

**Comment by rprouse** [comment]
Looks good now. A bunch of these changes would be too minor on their own, but since you did them as a batch 👍

**Pull request merged**